backend/business-services/egov-hrms/README.md

```
@@ -0,0 +1,112 @@
+- egov-filestore
+
+### Swagger API Contract
+- Please refer to the [Swagger API contarct](https://editor.swagger.io/?url=https://raw.githubusercontent.com/egovernments/business-services/master/Docs/hrms-v1.0.0.yaml#!/) for HRMS service to understand the structure of APIs and to have visualization of all internal APIs.
+
+
+## Service Details
+**Details of all the entities involved:** 
+
+**a) Assignments:** Every employee is assigned a list of assignments, every assignment is a designation provided to that employee for a given period of time. These designations are mapped to departments. This also includes marking the employee as HOD for that dept if needed. Employee can also provide information on who does he report to.
+   
+   - **Constraints:**
```

markdownlint-cli2 (0.17.2)

Unordered list indentation
Expected: 0; Actual: 3

(MD007, ul-indent)